src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml

```diff
@@ -34,20 +31,18 @@ Actors:
       OperationName: AdminCommand
       OperationCommand:
         enableSharding: *Database
+        primaryShard: "rs0"
```

Comment:
[q] Are the shardNames hard coded based on the number of shards? As in if we start a test cluster with N shards will the names be rs0, rs1, ... rsN ?

Reply:
Yep, unfortunately there is not a pre-existing programmatic way to simply get the shard names in genny. [Here more details](https://mongodb.slack.com/archives/GSTBKTER5/p1681917907945459)